natives.json

```diff
@@ -102240,107 +102240,107 @@
 	},
 	"_NAMESPACE71": {
 		"0x049D5C615BD38BAD": {
-			"name": "_0x049D5C615BD38BAD",
+			"name": "_SHOW_TOOLTIP",
 			"comment": "",
 			"params": [
 				{
-					"type": "Any",
+					"type": "struct",
 					"name": "p0"
 				},
 				{
-					"type": "Any",
+					"type": "struct",
 					"name": "p1"
 				},
 				{
-					"type": "Any",
+					"type": "int",
 					"name": "p2"
 				}
 			],
 			"return_type": "Any",
 			"build": "1207"
 		},
 		"0xD05590C1AB38F068": {
-			"name": "_0xD05590C1AB38F068",
+			"name": "_SHOW_TOWN_NOTIFICATION",
 			"comment": "",
 			"params": [
 				{
-					"type": "Any",
+					"type": "struct",
 					"name": "p0"
 				},
 				{
-					"type": "Any",
+					"type": "struct",
 					"name": "p1"
 				},
 				{
-					"type": "Any",
+					"type": "int",
 					"name": "p2"
 				},
 				{
-					"type": "Any",
+					"type": "int",
 					"name": "p3"
 				}
 			],
 			"return_type": "Any",
 			"build": "1207"
 		},
 		"0xCEDBF17EFCC0E4A4": {
-			"name": "_0xCEDBF17EFCC0E4A4",
+			"name": "_SHOW_OBJECTIF",
 			"comment": "",
 			"params": [
 				{
-					"type": "Any",
+					"type": "struct",
 					"name": "p0"
 				},
 				{
-					"type": "Any",
+					"type": "struct",
 					"name": "p1"
 				},
 				{
-					"type": "Any",
+					"type": "int",
 					"name": "p2"
 				}
 			],
 			"return_type": "Any",
 			"build": "1207"
 		},
 		"0xB2920B9760F0F36B": {
-			"name": "_0xB2920B9760F0F36B",
+			"name": "_SHOW_SIMPLE_RIGHT_TEXT",
```

Comment:
What is a "simple right text"?.
Either add a comment describing it or think of a better name.